src/bin/pgcopydb/cli_common.c

```diff
@@ -653,8 +681,7 @@ cli_copy_db_getopts(int argc, char **argv)
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
-	const char *optstring =
-		"S:T:D:J:I:b:L:u:mcAPOXj:xBeMlUagkynF:F:Q:irRCN:fp:ws:o:tE:Vvdzqh";
```

Comment:
If we generate the optstring value for clone command at run-time, we get `"S:T:D:J:J:I:b:L:L:u:mcAPOXj:xBBeMMlUagkynF:F:Q:irRCN:fp:ws:o:tE:Vvvdzqh"`. 

This character array contains duplicate characters due to the aliases that we have, and works fine.

Another thing that I noticed is that we had a substring `F:F:` that also repeated a short option string.

Reply:
Is there a way to remove duplicates in the code that builds the string here?

Reply:
I added deduplication logic. However to make things simple, I assumed the aliases will always be consecutive in the long_options array.

Reply:
Maybe we could use our hash lib to handle that fully, and hide that behind a DEBUG flag so that we don't do the dedup operation at run-time in “production” builds?

Reply:
Good idea. I think I can take this idea and build some more on top of it to come up with a nice design.

I will rewrite the helper function in a way that:
- on prod: it will immediately return supplied static strings
- on debug builds: run dedup logic, and verify that the result is identical to the supplied static string. If they are not identical, it will result in an assertion failure that should guide the developer into how they should update the string.

This way, we will be able to catch potential bugs in the future as CI will catch those assertions